administration/src/bp-modules/cards/ImportCardsFromFreinetController.ts

```diff
@@ -0,0 +1,66 @@
+import { BAVARIA_CARD_TYPE_GOLD, BAVARIA_CARD_TYPE_STANDARD } from '../../cards/extensions/BavariaCardTypeExtension'
+import { ProjectConfig } from '../../project-configs/getProjectConfig'
+
+const FREINET_EXPIRATION_DATE_COLUMN_HEADER = 'eak_datum'
+const FREINET_FIRSTNAME_COLUMN_HEADER = 'vorname'
+const FREINET_LASTNAME_COLUMN_HEADER = 'nachname'
+const FREINET_CARDTYPE_COLUMN_HEADER = 'inhaber_ehrenamtskarte'
+
+const mergeFirstAndLastnameIntoNewColumn = (line: string[], csvHeader: string[], nameColumnName: string) => {
+  const columnNameDoesNotExist = csvHeader.indexOf(nameColumnName) == -1
+  if (columnNameDoesNotExist) {
+    csvHeader[csvHeader.length] = nameColumnName
+  }
+
+  const indexVorname = csvHeader.indexOf(FREINET_FIRSTNAME_COLUMN_HEADER)
+  const indexNachname = csvHeader.indexOf(FREINET_LASTNAME_COLUMN_HEADER)
+  const fullname = `${line[indexVorname] ?? ''} ${line[indexNachname] ?? ''}`
+  line[csvHeader.indexOf(nameColumnName)] = fullname.trim()
+}
+
+const renameExpirationDateHeader = (csvHeader: string[], expiryColumnName: string) => {
+  const indexExpirationDate = csvHeader.indexOf(FREINET_EXPIRATION_DATE_COLUMN_HEADER)
+  if (indexExpirationDate != -1) {
```

Comment:
Is the linting somehow broken? Or is it just configured way less strict?
```suggestion
  if (indexExpirationDate !== -1) {
```
A few more below.